docs/release_notes/release_notes-0.16.0.rst

```diff
@@ -12,7 +12,7 @@ Key Features and Updates
 * Benchmarking enhancements
   *
 * Refactor Codebase
-  *
+  * REFACTOR-#4534: Remove logging metaclass (#4535)
```

Comment:
Could we maybe make this PR Name more descriptive? (i.e. `Replace logging meta class with class decorator` so that way it doesn't seem like we're rolling back logging capability?